src/cljs/proton/layers/tools/git/core.cljs

```diff
@@ -35,3 +35,4 @@
 (defmethod get-keybindings :git [] (get-keybindings :tools/git))
 (defmethod get-initial-config :git [] (get-initial-config :tools/git))
 (defmethod init-layer! :git [] (init-layer! :tools/git))
+(defmethod describe-mode :git [] {})
```

Comment:
We need to add the multimethod for `:tools/git` here. Changing this code to:
```
(defmethod describe-mode :git [] (describe-mode :tools/git))
```

Will make sure that backwards compatibility to the first version exists